surrogates/youtube-iframe-api.js

```diff
@@ -31,8 +31,13 @@
     // Mappings between the "real" video elements and their placeholder
     // elements.
     const videoElementsByID = new Map();
-    const videoElementByPlaceholderElement = new WeakMap();
-    const placeholderElementByVideoElement = new WeakMap();
+    const videoElementByPlaceholderElement = new Map();
+    const placeholderElementByVideoElement = new Map();
+
+    function* allVideoElements () {
```

Comment:
Returns an iterator that combines the two Map iterators. Slightly more efficient in this case than creating arrays from the iterators and concatenating those.